3.3/examples/django/README.md

```diff
@@ -0,0 +1,82 @@
+Django application example
+==========================
+
+This is a basic [Django](https://www.djangoproject.com/) project and some helper scripts to demonstrate how to deploy Python web applications with [OpenShift 3](http://www.openshift.org).
+
+We will be using Django 1.8 and the [PostgreSQL](http://www.postgresql.org/) database.
+
+OpenShift supports multiple deployment strategies. This example uses the Source-To-Image strategy, and the [Python 3.3 STI image](https://github.com/openshift/sti-python/tree/master/3.3) for producing application builds.
+
+**WARNING**: this is a _proof of concept_ example and, therefore, the code available here is not meant to be used in production.
+
+
+Application Build, Deploy, and Update Flow
+------------------------------------------
+
+This section assumes that you have built the `openshift` binary and generally have OpenShift 3 ready to go. See https://github.com/openshift/origin/blob/master/CONTRIBUTING.adoc.
+
+Start by cloning this repository and changing your current directory appropriately:
+
+```bash
+git clone https://github.com/openshift/sti-python.git
+cd sti-python/3.3/examples/django
+```
+
+### Start OpenShift ###
+
+To get you up and running quickly, you can run the `bootstrap.sh` script:
+
+```bash
+./bootstrap.sh
+```
+
+This will start OpenShift, create a private Docker registry, create a demo project and etc...
+
+OpenShift-speficic settings are defined in `application-template-stibuild.json`.
+
+
+### Building your app ###
+
+An initial build of your app is triggered because of ....
+
+You can see it with this command:
+
+```bash
+osc get builds
+```
+
+The STI build process essentially copies your source files and install dependencies listed in `requirements.txt`, finally producing a new Docker image.
+
+
+### Application deployment ###
+
+An application deployment is triggered right after a build because of ...
+
+Your database pod was deployed because of ...
+
+You should now the able to see your Django application running at: http://<host>:<ip>
+
+
+### Updates and redeploys ###
+
+A build can be triggered automatically via a Github push hook, or manually with:
+
+```bash
+osc start-build django-demo-build
+```
+
+### Bonus: osc edit ###
+
+The `osc edit` command let's you change entities on-the-fly and have the new definitions reflected in your running cluster.
+
+You can use that to change the number of replicas, the URL of the git repository of your application, etc.
+
+
+Cleaning up
+-----------
+
+You can use the `cleanup.sh` script to clean up your local environment after running this demo:
+
+```bash
+./cleanup.sh
+```
```

Comment:
Generally speaking I'd focus inside of this file entirely on running your sample application inside of openshift, leaving out all the usual general OS details to OS's sample-app.